CodingChallenges/CC_075_Wikipedia/index.html

```diff
@@ -17,7 +17,7 @@
 	<h2>Wikipedia Coding Challenge Thing</h2>
 
 	<p>
-		word: <input id="userinput" value="rainbow"></input>
+		word: <input id="userinput" value="rainbow" />
```

Comment:
Oh, so that's how you do it!

Reply:
Technically in html5 you can also leave them as just `<input id="userinput" value="rainbow">`, but this style is more generally compatible (older html, xhtml, etc.)